bundles/org.openhab.binding.kaleidescape/src/main/java/org/openhab/binding/kaleidescape/internal/handler/KaleidescapeHandler.java

```diff
@@ -0,0 +1,603 @@
+                            }
+
+                            // only Strato supports the GET_*_COLOR commands
+                            if (STRATO.equals(this.componentType)) {
+                                initialCommands.addAll(
+                                        new ArrayList<String>(Arrays.asList(GET_VIDEO_COLOR, GET_CONTENT_COLOR)));
+                            }
+
+                            initialCommands.forEach(command -> {
+                                try {
+                                    connector.sendCommand(command);
+                                } catch (KaleidescapeException e) {
+                                    logger.debug("{}: {}", "Error sending initial commands", e.getMessage());
+                                }
+                            });
+
+                            if (this.updatePeriod == 1) {
+                                connector.sendCommand(SET_STATUS_CUE_PERIOD_1);
+                            }
+                        } catch (KaleidescapeException e) {
+                            error = "First command after connection failed";
+                            logger.debug("{}: {}", error, e.getMessage());
+                            closeConnection();
+                        }
+                    } else {
+                        error = "Reconnection failed";
+                    }
+                    if (!error.equals(EMPTY)) {
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, error);
+                        return;
+                    }
+                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE, this.friendlyName);
+                    lastEventReceived = System.currentTimeMillis();
+                }
+            }
+        }, 1, RECON_POLLING_INTERVAL, TimeUnit.SECONDS);
+    }
+
+    /**
+     * Cancel the reconnection job
+     */
+    private void cancelReconnectJob() {
+        ScheduledFuture<?> reconnectJob = this.reconnectJob;
+        if (reconnectJob != null) {
+            reconnectJob.cancel(true);
+            this.reconnectJob = null;
+        }
+    }
+
+    /**
+     * Schedule the polling job
+     */
+    private void schedulePollingJob() {
+        logger.debug("Schedule polling job");
+        cancelPollingJob();
+
+        pollingJob = scheduler.scheduleWithFixedDelay(() -> {
+            synchronized (sequenceLock) {
+                if (connector.isConnected()) {
+                    logger.debug("Polling the component for updated status...");
+                    try {
+                        connector.ping();
+                    } catch (KaleidescapeException e) {
+                        logger.debug("Polling error: {}", e.getMessage());
+                    }
+
+                    // if the last successful polling update was more than 1.25 intervals ago,
+                    // the component is not responding even though the connection is still good
```

Comment:
The device is very responsive and the polling just asks it to return the current time (as recommended by the control protocol specs). So I think this will be OK.